common/layerSharedLogic.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+import * as crypto from "crypto";
+import log from "loglevel";
+import {
+  runtimeToLayerName,
+  DD_GOV_ACCOUNT_ID,
+  DD_ACCOUNT_ID,
+  LAYER_PREFIX,
+  EXTENSION_LAYER_PREFIX,
+} from "./constants";
+
+export function getLambdaLayerArn(region: string, version: number, runtime: string, isArm: boolean, isNode: boolean) {
+  const baseLayerName = runtimeToLayerName[runtime];
+  const layerName = isArm && !isNode ? `${baseLayerName}-ARM` : baseLayerName;
+  // TODO: edge case where gov cloud is the region, but they are using a token so we can't resolve it.
+  const isGovCloud = region === "us-gov-east-1" || region === "us-gov-west-1";
```

Comment:
since those constants are used in multiple places, maybe we should define it somewhere (maybe as an array so we could loop over it, and if a new region is added, the change will be done everywhere it's used?)

Reply:
Are you referring to the two gov-cloud regions?

Reply:
yes! sorry if that was unclear

Reply:
fixed! used a readonly array constant in `constants.ts` and using `Array.includes()` instead of hardcoding the two checks